src/App.jsx

```diff
@@ -14,57 +15,92 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [good, setGood] = useState('Jam');
+  const [active, setActive] = useState(
```

Comment:
Actually this state looks redundant here, you can use your state above and compare good with current item inside map function

Reply:
ty